scripts/community/achievements.js

```diff
@@ -120,21 +120,34 @@ function InitAchievements( items, isPersonal )
 
 	extraTabs.append( CreateFoldButton( true ) );
 
-	if( window.innerWidth < 600 )
+	function AppendExtraTabs()
 	{
-		extraTabs.classList.add( 'steamdb_stats_extra_tabs_mobile' );
-		document.querySelector( '#subtabs' ).insertAdjacentElement( 'afterend', extraTabs );
-	}
-	else
-	{
-		document.querySelector( '#tabs' ).append( extraTabs );
+		if( window.innerWidth < 600 )
+		{
+			extraTabs.classList.add( 'steamdb_stats_extra_tabs_mobile' );
+			document
```

Comment:
If this is to be run on every resize, it should at least check that the element isn't already where it should be, recreating it is not good.